core/base/entry.go

```diff
@@ -29,8 +29,11 @@ func (e *SentinelEntry) SetError(err error) {
 	}
 }
 
-func (e *SentinelEntry) Context() *EntryContext {
```

Comment:
From my perspective, export Context don't make sense.
1. Context should be sentinel internal status,
2. Context is maintained by pool, exporting might cause fatal bug.